catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/utils/BrickTestUtils.java

```diff
@@ -84,4 +98,36 @@ public static void createUserListFromDataFragment(String userListName, boolean f
 		onView(withText(userListName))
 				.check(matches(isDisplayed()));
 	}
+
+	public static void addBrick(Class<?> brickHeaderClass, int brickCategoryId) {
+		onView(withId(R.id.button_add))
+				.perform(click());
+		onData(allOf(Is.is(instanceOf(String.class)), Is.is(UiTestUtils.getResourcesString(brickCategoryId))))
+				.inAdapterView(BrickCategoryListMatchers.isBrickCategoryView())
+				.perform(click());
+		onData(Is.is(instanceOf(brickHeaderClass))).inAdapterView(BrickPrototypeListMatchers.isBrickPrototypeView())
+				.perform(click());
+
+		onData(instanceOf(Brick.class))
+				.inAdapterView(ScriptListMatchers.isScriptListView()).atPosition(1)
+				.perform(click());
+	}
+
+	public static void deleteItemWithTextFromFormulaEditorDataList(String text) {
+		onView(withId(R.id.formula_editor_keyboard_data))
+				.perform(click());
+		onView(allOf(withText(text), withId(R.id
+				.fragment_formula_editor_datalist_item_name_text_view)))
+				.perform(longClick());
+		onView(withText(R.string.delete))
+				.perform(click());
+		try {
```

Comment:
i dont like try / catches in test code. especially in utility methods, where it can potentially hide errors.
another PR.
ive seen something similar also in another PR, and they solved it via additional parameter.

Reply:
ok, then I'll do it via an additional parameter